addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -550,7 +550,7 @@ var FileWidget = SearchableMediaWidget.extend({
         this.$addUrlButton.toggleClass('btn-secondary', emptyValue)
             .toggleClass('btn-primary', !emptyValue)
             .prop('disabled', !isURL);
-        this.$urlSuccess.toggleClass('d-none', !isURL);
+        this.$urlSuccess.toggleClass('d-none', !isURL || emptyValue);
```

Comment:
I don't understsand the link between the specifications for the task and this particular change.